Week2/homework/js-exercises/recipeCard.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+const myRecipe = {
+mealName: 'Ash', 
+ servings: 2,
+    Ingredients:
+        ['250g meat',
+            'grocery',
+            '500g flour']
+};
+for (const pro in myRecipe) {
```

Comment:
can you use a more clear variable name then `pro` :)?

Reply:
it was property, i thought it is short name of that.

Reply:
changed